frontend/src/lang/en.ts

```diff
@@ -368,6 +371,7 @@ export const en = {
       },
       tos: {
         header: "Terms of service",
+        title: "Terms of service",
```

Comment:
@zdeveloper pointed out in some pair review that we use identical phrases for the english translation but the header and the title are different for the spanish.

Not sure who to follow up with on this 🤷‍♂️

Reply:
Ooh, that's a good catch. Maybe @lauraykerr can adjudicate, or knows who can